scripts/lua/find_host.lua

```diff
@@ -25,10 +25,30 @@ print [[
       res = interface.findHost(query)
 
       if(res ~= nil) then
+	 values = {}
 	 for k, v in pairs(res) do
-	    if(v ~= "") then 
+	    if(v ~= "") then
+	       if not values[v] then
+	          values[v] = 1
+	       else
+	          values[v] = values[v] + 1
+	       end
+	    end
+	 end
+	 
+	 for k, v in pairs(res) do
+	    if(v ~= "") then
+	       if values[v] > 1 then
+	          -- we matched both an ipv4 and ipv6 with same host name, display differently
+	          if findString(v, ":") == nil then
```

Comment:
check lua_utils there are already functions that check if the IP is v4 or v6 and they should also be more accurate